grpc.go

```diff
@@ -1634,7 +1634,7 @@ func (a *agentGRPC) CopyFile(ctx context.Context, req *pb.CopyFileRequest) (*gpb
 	// container's rootfs is mounted at /run, in order to avoid overwrite guest's rootfs files, only
 	// is possible to copy files to /run
 	if !strings.HasPrefix(path, containersRootfsPath) {
-		return emptyResp, fmt.Errorf("Only is possible to copy files into the %s directory", containersRootfsPath)
+		agentLog.Info("Copying files to guest's rootfs")
```

Comment:
I think we'd better get @devimc's comments on this behavioral change.

Reply:
coz, it feels like a potential big security hole to me... which I suspect is why we restrict the destination path current;y.
@nitkon ooi, why do you need to set the VM hostname etc.??
/cc @amshinde

Reply:
@nitkon Same questions as @grahamwhaley. 
Why do you need to set the VM hostname? Can you explain the use-case?

Reply:
Can you copy file to `/run` and then bind mount it to `/etc/hostname` in `CreateSandbox` request if necessary?

Reply:
@bergwolf : Ptal ...  I have copied the file in `kataShared` on the host and then bind mounting it in the sandbox `/etc/resolv.conf` .